ibis_omniscidb/client.py

```diff
@@ -361,7 +295,7 @@ def read_csv(
         quotechar: Optional[str] = '"',
         delimiter: Optional[str] = ',',
         threads: Optional[int] = None,
-    ) -> OmniSciDBQuery:
+    ):
```

Comment:
any reason for removing the type annotation?

Reply:
I got it, thanks for the explanation!

it seems that geopandas.GeoDataFrame is an instance of pandas.DataFrame.

I was playing a bit with that and I have an example of a workaround:

```python

from __future__ import annotations
import pandas as pd
import geopandas as gpd

try:
    import cudf
except (ImportError, OSError):
    cudf = pd


def check_df(df: pd.DataFrame | cudf.DataFrame ) -> bool:
    return isinstance(df, (pd.DataFrame, cudf.DataFrame))


if __name__ == "__main__":
    assert check_df(pd.DataFrame())
    assert check_df(gpd.GeoDataFrame())
    assert check_df(cudf.DataFrame())
```

maybe it is not the best way to do that .. it is just an idea.

that is not a priority but, I think it would be good to have an issue for that.